tests/system/small/test_progress_bar.py

```diff
@@ -52,14 +52,6 @@ def test_progress_bar_scalar(penguins_df_default_index: bf.dataframe.DataFrame,
     assert_loading_msg_exist(capsys.readouterr().out)
 
 
-def test_progress_bar_read_gbq(session: bf.Session, penguins_table_id: str, capsys):
```

Comment:
Presumably this was causing us some trouble because of modifying global state?

Could you instead mark the test with pytest skip and add a TODO to re-enable once we add thread local context managers (internal issue 308657813)

Reply:
read_gbq doesn't copy the table anymore, so no loading messages either

Reply:
Ah, good point! Woohoo! 🎉